fe/fe-core/src/main/java/com/starrocks/clone/DiskAndTabletLoadReBalancer.java

```diff
@@ -927,7 +927,7 @@ private Map<Pair<Long, Long>, Set<Long>> getPartitionTablets(long beId, TStorage
 
             if (pathHash != -1) {
                 Replica replica = GlobalStateMgr.getCurrentInvertedIndex().getReplica(tabletId, beId);
-                if (replica.getPathHash() != pathHash) {
+                if (replica == null || replica.getPathHash() != pathHash) {
```

Comment:
can the replica be null, we don't have the db lock hold here?

Reply:
No db lock hold here, the metadata is retrieved only from TabletInvertedIndex.